### DISCUSSION TITLE
TPMS disable error

### DISCUSSION THREAD

**Discussion by DavidIgor** [cedricp/ddt4all#1072] [Q&A]
![image](https://github.com/cedricp/ddt4all/assets/122531259/32964a4b-0706-499a-b846-07a652d4163a)
when I set TPMS to "without TPMS" it shows this on display and nothing is available. ironically tpms warning light goes away ...

**Comment by AndyIreland** [comment]
What car is that - looks like a dacia? - is it a Duster? 

If you disabled TPMS from the correct ECU(S) - have you turned off TPMS in the Cluster ECU?
![image](https://github.com/cedricp/ddt4all/assets/49172217/d7abafd9-0175-4d94-b502-a517b390f8af)

**Comment by DavidIgor** [comment]
Hi @AndyIreland it is dokker and I have done it only in cluster (as per your photo). I was looking for another option in other ecus but can't find tpms.(only in cluster)

**Comment by matzel82** [comment]
Go to the BCM, TPMS .... there you have config SSPP without

**Comment by AndyIreland** [comment]
> Hi @AndyIreland it is dokker and I have done it only in cluster (as per your photo). I was looking for another option in other ecus but can't find tpms.(only in cluster)

Yes you need to disable it in BCM/UCH ecu as well

**Comment by DavidIgor** [comment]
with scan option ill get just these ecus:
![image](https://github.com/cedricp/ddt4all/assets/122531259/c288ecc7-ba97-4031-8bb6-999209535dae)

**Comment by matzel82** [comment]
BCM....

**Comment by DavidIgor** [comment]
will check tomorrow, but I haven't seen anything tpms related in there ...  also why is it red? (I do have elm327 with can switch to read multimedia ecus ...

**Comment by matzel82** [comment]
![image](https://github.com/cedricp/ddt4all/assets/31157216/e37c4513-ee76-4970-ae58-b3383abb8625)

take a look here please. Is there a „with“ change it to „without“

Another one I haven't found yet

**Comment by DavidIgor** [comment]
right! all sorted you were right. thanks for help
